src/main/core/support/options.c

```diff
@@ -172,8 +169,6 @@ Options* options_new(gint argc, gchar* argv[]) {
       { "socket-recv-buffer", 0, 0, G_OPTION_ARG_INT, &(options->initialSocketReceiveBufferSize), sockrecv->str, "N" },
       { "socket-send-buffer", 0, 0, G_OPTION_ARG_INT, &(options->initialSocketSendBufferSize), socksend->str, "N" },
       { "tcp-congestion-control", 0, 0, G_OPTION_ARG_STRING, &(options->tcpCongestionControl), "Congestion control algorithm to use for TCP ('aimd', 'reno', 'cubic') ['reno']", "TCPCC" },
-      { "tcp-ssthresh", 0, 0, G_OPTION_ARG_INT, &(options->tcpSlowStartThreshold), "Set TCP ssthresh value instead of discovering it via packet loss or hystart [0]", "N" },
-      { "tcp-windows", 0, 0, G_OPTION_ARG_INT, &(options->initialTCPWindow), "Initialize the TCP send, receive, and congestion windows to N packets [10]", "N" },
```

Comment:
The [getting started tutorial doc](https://github.com/shadow/shadow/blob/dev/docs/2-Getting-Started-Tutorial.md#example-experiment) has an example that tweaks the `--tcp-windows` option. We should update that example experiment so it no longer references a non-existent option.

Reply:
I updated this example to use different queuing disciplines instead. Let me know if you think there's a better option to use.